### PR TITLE
added content desktop browser

### PR DESCRIPTION
**Pull request by cwi83**
added description how to start a meeting withe the desktop browser

### CODE REVIEW

File: docs/user-guide/start-a-jitsi-meeting.md [hunk 1]
```diff
@@ -4,6 +4,26 @@ title: Start a Jitsi Meeting
 sidebar_label: Start a Jitsi Meeting
 ---
 
-Welcome to the user guide!
+## Desktop Browser
 
-Check back soon!
+1. You need a browser (ideally Chrome from Google, please note our separate information)
```
Comment (saghul) [inline docs/user-guide/start-a-jitsi-meeting.md:9]: Leave the browser recommendation out please.

File: docs/user-guide/start-a-jitsi-meeting.md [hunk 2]
```diff
@@ -4,6 +4,26 @@ title: Start a Jitsi Meeting
...
+
+![screenshot 1](../assets/user_guide_join_meeting.png "screenshot 1")
+
+7. It is possible that no picture of you will appear at first. To do this, the browser will ask you whether you want to allow camera access. Please confirm this by clicking on "Allow" or "Allow". Sometimes you also have to click the camera button at the bottom of the screen first to activate the dialog for allowing camera access. Do the same with the microphone the first time you use Jitsi.
+8. Now enter your display name in the "enter your name" field.
+9. Click the blue "Join meeting" button.
+10. Have fun in your first conference.
+
+## Mobile Browser
+
```
Comment (saghul) [inline docs/user-guide/start-a-jitsi-meeting.md:29]: add the come back soon text here please.

**Review by saghul — CHANGES_REQUESTED** [review]
Thanks for getting this started! Left some comments.

File: docs/user-guide/start-a-jitsi-meeting.md (hunk 1 above)
Comment (cwi83) [reply docs/user-guide/start-a-jitsi-meeting.md]: i will do

File: docs/user-guide/start-a-jitsi-meeting.md (hunk 2 above)
Comment (cwi83) [reply docs/user-guide/start-a-jitsi-meeting.md:29]: i will do

**Comment by cwi83** [comment]
done :)

**Review by saghul — APPROVED** [review]
Thank you!

**Comment by cwi83** [comment]
yes, need some free time

**Comment by saghul** [comment]
Thank you!

**Pull request merged**